### DISCUSSION TITLE
vector search in id scope

### DISCUSSION THREAD

**Discussion by winterfellding** [milvus-io/milvus#4927] [Q&A and General discussion]
Hi, I'm using Milvus v1.0 to do image search, I have a feature to implement, each image have some attribute, e.g, some image's author is A, and some other is B, does milvus now support search all author A's image and rank with relevance.

I notice that v1.0 now doesn't support property search for the vector, can I save the milvus vector id and other properties to filter in another storage system, and use an id collection as the search filter in milvus search.

**Comment by shengjun1985** [comment]
The current v1.x does not support attribute filtering. And in the future, v2.x may support this feature.
Do you have many kinds of attributes?
If not, you can insert images with different attributes into different partitions, and then query in the specific partition.

**Comment by yhmo** [comment]
In some user applications, they use other SQL system(mysql, postgresql, etc.) to store properties.
The next major version (milvus v2.0) will support attribute filtering search.
